pkg/apply/prune/prune.go

```diff
@@ -137,7 +137,7 @@ func (p *Pruner) Prune(
 				if errors.As(filterErr, &abandonErr) {
 					if !opts.DryRunStrategy.ClientOrServerDryRun() {
 						var err error
-						obj, err = p.removeInventoryAnnotation(obj)
+						obj, err = p.removeInventoryAnnotation(taskContext.Context(), obj)
 						if err != nil {
```

Comment:
Maybe move the taskContext to the first method arg

Reply:
done